front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/ActiveUsersWidget/ActiveUsersCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+
+// hooks
+import useActiveUsers from 'modules/commercial/analytics/admin/components/ActiveUsersCard/useActiveUsers';
+
+// components
+import { Box, Title } from '@citizenlab/cl2-component-library';
+import Chart from 'modules/commercial/analytics/admin/components/ActiveUsersCard/Chart';
```

Comment:
Again this Chart will need refactoring to move into components/admin/GraphCards/ActiveUsersCard/